src/app-nordigen/services/nordigen-service.js

```diff
@@ -17,8 +17,8 @@ import config from '../../load-config.js';
 
 const NordigenClient = nordigenNode.default;
 const nordigenClient = new NordigenClient({
-  secretId: config.nordigen_secret_id,
-  secretKey: config.nordigen_secret_key,
+  secretId: config.nordigen.secretId,
+  secretKey: config.nordigen.secretKey,
 });
```

Comment:
This should probably not be set up at all if the config is not provided.

Reply:
I agree. Something we should fix as a follow-up. I'll add it to the list: https://github.com/actualbudget/actual/issues/724

Reply:
(Fine to defer to a future PR tho)